share/metainfo/virtaal.metainfo.xml.in

```diff
@@ -41,6 +41,6 @@
   </screenshot>
  </screenshots>
  <url type="homepage">http://virtaal.org/</url>
```

Comment:
How about use this to update the homepage to https://virtaal.translatehouse.org/ ?